### PR TITLE
Cleanup more use of the "long" data type.

### PR DESCRIPTION
**Pull request by OmniBlade**
This datatype varies in length between platforms so its use is not portable.

### CODE REVIEW

File: redalert/conquer.cpp [hunk 1]
```diff
@@ -4257,7 +4257,7 @@ bool Force_CD_Available(int cd)
 #endif
 
 #ifdef FRENCH
-                sprintf(buffer, "Ins�rez le %s", _cd_name[cd]);
+                sprintf(buffer, "Ins�rez le %s", _cd_name[cd]);
```
Comment (giulianobelinassi) [inline redalert/conquer.cpp]: Looks like this change was done by mistake.

File: redalert/conquer.cpp [hunk 2]
```diff
@@ -4268,7 +4268,7 @@ bool Force_CD_Available(int cd)
             } else {
 #ifdef DVD
 #ifdef FRENCH
-                sprintf(buffer, "Ins�rez le %s", _cd_name[4]);
+                sprintf(buffer, "Ins�rez le %s", _cd_name[4]);
```
Comment (giulianobelinassi) [inline redalert/conquer.cpp]: Looks like this change was done by mistake.

File: redalert/jshell.cpp [hunk 3]
```diff
@@ -282,7 +282,7 @@ int Load_Picture(char const* filename,
  * HISTORY:                                                                                    *
  *   04/02/1994 JLB : Created.                                                                 *
  *=============================================================================================*/
-long Translucent_Table_Size(int count)
+int Translucent_Table_Size(int count)
 {
     return (256L + (256L * count));
```
Comment (giulianobelinassi) [inline redalert/jshell.cpp]: Still long constants

File: redalert/netdlg.cpp [hunk 4]
```diff
@@ -5389,31 +5389,31 @@ void Log_Message(char* msg)
...
- *    �     [Reject]             Count:--- ##      �                                           *
- *    �                          Level:--- ##      �                                           *
- *    �                                            �                                           *
- *    �               Credits: _____               �                                           *
- *    �       [  Bases   ]   [   Crates   ]        �                                           *
- *    �       [ Tiberium ]   [ AI Players ]        �                                           *
- *    �  �������������������������������������Ŀ   �                                           *
- *    �  �                                     �   �                                           *
- *    �  �                                     �   �                                           *
- *    �  ���������������������������������������   �                                           *
```
Comment (giulianobelinassi) [inline redalert/netdlg.cpp]: Looks like this change was done by mistake.

File: tiberiandawn/jshell.cpp [hunk 5]
```diff
@@ -283,7 +283,7 @@ long Load_Uncompress(FileClass& file, BufferClass& uncomp_buff, BufferClass& des
  * HISTORY:                                                                                    *
  *   04/02/1994 JLB : Created.                                                                 *
  *=============================================================================================*/
-long Translucent_Table_Size(int count)
+int Translucent_Table_Size(int count)
 {
     return (256L + (256L * count));
```
Comment (giulianobelinassi) [inline tiberiandawn/jshell.cpp]: Still long constants.

File: tiberiandawn/logic.cpp [hunk 6]
```diff
@@ -74,20 +74,20 @@ void LogicClass::Debug_Dump(MonoClass* mono) const
...
-                "�Factories.�   �                                         �\r"
-                "�          �   �                                         �\r"
-                "�          �   �                                         �\r"
-                "����������������������������ĴSpare CPU Time��������������\r");
+    mono->Print("��������������������������������������������������������Ŀ\r"
+                "�Units.....�   �Frame Rate:      Avg:      Frame:        �\r"
+                "�Infantry..�   �����������������������������������������Ĵ\r"
+                "�Aircraft..�   �                                         �\r"
+                "�Buildings.�   �                                         �\r"
+                "�Terrain...�   �                                         �\r"
```
Comment (giulianobelinassi) [inline tiberiandawn/logic.cpp]: Looks like this change was done by mistake.

**Review by giulianobelinassi — CHANGES_REQUESTED** [review]

**Comment by giulianobelinassi** [comment]
There are changes which seems to have been done by mistake automatically by your text editor.

**Comment by OmniBlade** [comment]
Should all be cleaned up now.

File: redalert/conquer.cpp [hunk 7]
```diff
@@ -3540,8 +3540,8 @@ void Handle_Team(int team, int action)
     **	Create the team.
     */
     case 2: {
-        long minx = 0x7FFFFFFFL, miny = 0x7FFFFFFFL;
-        long maxx = 0, maxy = 0;
+        int minx = 0x7FFFFFFFL, miny = 0x7FFFFFFFL;
```
Comment (giulianobelinassi) [inline redalert/conquer.cpp]: long constant

File: redalert/dllinterface.cpp [hunk 8]
```diff
@@ -7568,8 +7568,8 @@ void DLLExportClass::Team_Units_Formation_Toggle_On(uint64 player_id)
     //
 
     int team = MAX_TEAMS;
-    long minx = 0x7FFFFFFFL, miny = 0x7FFFFFFFL;
-    long maxx = 0, maxy = 0;
+    int minx = 0x7FFFFFFFL, miny = 0x7FFFFFFFL;
```
Comment (giulianobelinassi) [inline redalert/dllinterface.cpp]: long constant

File: tiberiandawn/aircraft.cpp [hunk 9]
```diff
@@ -2000,26 +2000,26 @@ void AircraftClass::Debug_Dump(MonoClass* mono) const
...
-                "�Recoiling.....� � �                                                           \n"
-                "�To Display....� � �                                                           \n"
-                "��������������������                                                           \n");
+    mono->Print("�Name:���������������Mission:����TarCom:�NavCom:�Radio:�Coord:���Altitude�St:Ŀ\n"
+                "�                   �           �       �       �      �        �        �    �\n"
+                "����������������N�Y�Health:��Fdir:��Bdir:��Speed:��������������Cargo:��������Ĵ\n"
+                "�Active........� � �        �     �       �      �            �               �\n"
+                "�Limbo.........� � ����������������������������������������������������������Ĵ\n"
+                "�Owned.........� � �Last Message:                                             �\n"
+                "�Discovered....� � �Timer:�Arm:������������������Flash:�Stage:�Team:�����Arch:�\n"
```
Comment (giulianobelinassi) [inline tiberiandawn/aircraft.cpp]: Mistake change

File: tiberiandawn/function.h [hunk 10]
```diff
@@ -39,53 +39,53 @@
 Map(screen) class heirarchy.
 
     MapeditClass(most derived class)-- scenario editor
-        � MouseClass-- handles mouse animation and display control
-        � ScrollClass-- map scroll handler
-        � HelpClass-- pop
+        � MouseClass-- handles mouse animation and display control
+        � ScrollClass-- map scroll handler
```
Comment (giulianobelinassi) [inline tiberiandawn/function.h]: mistake change

File: tiberiandawn/function.h [hunk 11]
```diff
@@ -39,53 +39,53 @@
...
+        � HelpClass-- pop
     - up help text handler
-        � TabClass-- file folder tab screen mode control dispatcher
-        � SidebarClass-- displays and controls construction list sidebar
-        � PowerClass-- display power production
+        � TabClass-- file folder tab screen mode control dispatcher
+        � SidebarClass-- displays and controls construction list sidebar
```
Comment (giulianobelinassi) [inline tiberiandawn/function.h]: mistake change

File: tiberiandawn/function.h [hunk 12]
```diff
@@ -39,53 +39,53 @@
...
+        � PowerClass-- display power production
           / consumption bargraph
-        � RadarClass-- displays and controls radar map
-        � DisplayClass-- general tactical map display handler
-        � MapClass-- general tactical map data handler
-        � GScreenClass(pure virtual base class)-- generic screen control
+        � RadarClass-- displays and controls radar map
+        � DisplayClass-- general tactical map display handler
+        � MapClass-- general tactical map data handler
```
Comment (giulianobelinassi) [inline tiberiandawn/function.h]: mistake change

File: tiberiandawn/function.h [hunk 13]
```diff
@@ -39,53 +39,53 @@
...
+                                  �
+                                  �
+                                  � ObjectClass
+                                  �
+       ����������������������������������������������������������������Ŀ
+          AnimClass  � TemplateClass    �        �� FuseClass     � TerrainClass
+              �                   �        �� FlyClass      �
+              �                   � BulletClass            � OverlayClass MissionClass SmudgeClass
+                                  � RadioClass
+                                  �
```
Comment (giulianobelinassi) [inline tiberiandawn/function.h]: mistake change

File: tiberiandawn/function.h [hunk 14]
```diff
@@ -39,53 +39,53 @@
...
-             � TemplateTypeClass         �
-    �����������������������������������������Ŀ TerrainTypeClass
-    �              �           �              � UnitTypeClass      � BuildingTypeClass      �
-                   � InfantryTypeClass AircraftTypeClass
+                                    � ObjectTypeClass
+                                    �
+             �����������������������������������������������������Ŀ
+             �                      �            �                 � TechnoTypeClass              �            �                 �
+             � BulletTypeClass    �                 �
+             � TemplateTypeClass         �
```
Comment (giulianobelinassi) [inline tiberiandawn/function.h]: mistake change

**Review by giulianobelinassi — CHANGES_REQUESTED** [review]

**Comment by OmniBlade** [comment]
Aw man, that stupid utf conversion :D

**Comment by OmniBlade** [comment]
Okay, really should be cleaned up this time @giulianobelinassi

File: redalert/conquer.cpp (hunk 1 above)
Comment (giulianobelinassi) [inline redalert/conquer.cpp]: Mistake change

File: redalert/conquer.cpp (hunk 2 above)
Comment (giulianobelinassi) [inline redalert/conquer.cpp]: Mistake change

**Review by giulianobelinassi — CHANGES_REQUESTED** [review]

File: redalert/conquer.cpp (hunk 1 above)
Comment (OmniBlade) [reply redalert/conquer.cpp]: I think these are actually fixing a change that went through incorrectly previously.

**Review by giulianobelinassi — APPROVED** [review]

**Pull request merged**